modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/status/ServiceStatusChecker.kt

```diff
@@ -32,7 +36,8 @@ class ServiceStatusChecker @Inject constructor(@ApplicationContext val context:
     private suspend fun checkUrl(url: String): ServiceStatus {
         val log = StringBuilder()
 
-        val okHttpClient = OkHttpClient.Builder()
+        val okHttpClient = httpClient
+            .newBuilder()
```

Comment:
I would like to avoid doing this as well but I don't know how we could access `chain.connection()` other than through an interceptor. And this way seems much simpler than synchronizing an interceptor and logs across multiple threads/coroutines.